forge-gui/res/editions/Scars of Mirrodin.txt

```diff
@@ -266,7 +265,7 @@ ScryfallCode=SOM
 1 w_2_2_cat @Scott Chou
 2 w_1_1_soldier @Goran Josic
 3 r_1_1_goblin @Goran Josic
-4 g_1_1_phyrexian_insect_infect @Adrian Smith
+4 g_1_1_insect_infect @Adrian Smith
```

Comment:
@Eradev if we do fallback to newer TokenName too, then i would say it could be approved